examples/kotlin/src/main/kotlin/com/example/authors/QueriesImpl.kt

```diff
@@ -36,91 +32,71 @@ ORDER BY name
 class QueriesImpl(private val conn: Connection) : Queries {
```

Comment:
Personally I try to avoid the `Impl` suffix because it doesn't say anything about the implementation. Maybe `GeneratedQueries`?

Reply:
But it’s the Java way! I agree. But since naming didn’t change in this PR I’d like to leave it for a follow up if you feel strongly? Also all of this code is generated. Maybe RealQueries which isn’t anymore meaningful but seems like the Kotlin way. 

I think you’ve been corrupted by Go

Reply:
The `Impl` suffix is afaict never used in the JDK so while it's common in some Java projects it's not really a "sanctioned" convention. :-) I don't care about it strongly but something to think about.